### PR TITLE
Epic/3939 run updated selenium image on travis

### PR DESCRIPTION
**Pull request by amoedoamorim**
## Description

This PR updates our selenium standalone chrome image and does a few tweaks to ensure stability.

It reduces the selenium server-side timeout (120s) from the image default (300s), increases the client-side timeout for new sessions (180s) and open more concurrent session slots so that tests don't fail sequentially with `Net::ReadTimeout` due to stuck not-cleaned up slots.

Due to the new chrome cookie handling it also requires that test scripts running on web container and the application running on the browser see everything served from the same domain, so we're adding nginx to the web container to make it all appear to be running on localhost from whichever container.

It also fixes the imgur upload, which was failing with "400: Bad Request". Changed the type to `base64` and it seems to work now.

H/T @caiosba @danielevalverde for the help taming this selenium monster

References: CV2-3939

## Type of change

- [ ] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Automated test (add or update automated tests)

## How has this been tested?

Tested the setup locally, manually starting the nginx proxies on web and chromedriver before running local integration tests.

## Things to pay attention to during code review

Please describe parts of the change that require extra attention during code review, for example:

- File FFFF, line LL: This refactoring does this and this. Is it consistent with how it’s implemented elsewhere?
- Etc.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [x] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [ ] If my components involve user interaction - specifically button, text fields, or other inputs - I have added a [BEM-like class name](https://meedan.atlassian.net/wiki/spaces/ENG/pages/1327628289/Naming+conventions+for+interactive+elements) to the element that is interacted with
- [ ] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

### CODE REVIEW

**Comment by caiosba** [comment]
I'm going to review it tomorrow, but for now I would like to send big kudos for the amazing work handling this beast, @amoedoamorim ! 👏🏽👏🏽👏🏽

File: .travis.yml [hunk 1]
```diff
@@ -51,8 +51,8 @@ before_install:
 - cp ../test/google_credentials.json google_credentials.json
 - cd -
 # Configuration files
-- cp test/config.js.example config.js
-- cp test/config.js.example test/config.js
+- cp config.js.example config.js
+- cp config.js.example test/config.js
```
Comment (caiosba) [inline .travis.yml:55]: Does it mean that we don't need the `example` files inside the `test/` directory? If so, can we delete them?

**Review by caiosba — APPROVED** [review]
Great work @amoedoamorim! :tada:

**Review by danielevalverde — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (amoedoamorim) [reply .travis.yml:55]: Hmm, we still do need `test/config.yml.example` and we might still use `test/config.js.example` if we're not running the nginx proxies locally, which I haven't made autostart for the local test env. It gets started automatically for Travis jobs.

Comment (caiosba) [reply .travis.yml:55]: OK sounds good!

**Pull request merged**